src/Rings/mpolyquo-localizations.jl

```diff
@@ -1381,7 +1381,7 @@ end
                    <:MPolyQuoLocRing{<:Any, <:Any, <:Any, <:Any,
                                      <:MPolyPowersOfElement}})
   A = domain(f)
-  R = base_ring(f)
+  R = base_ring(A)
```

Comment:
@HechtiDerLachs If this is an unrelated fix, should it then be in another PR, or does this affect the tests for this PR?

Reply:
This can stay. I didn't include specific tests for this, though. It just popped up in some local computation.